File: libs/api/src/main/java/com/example/api/graphapi/utils/FbUtils.java
package com.example.api.graphapi.utils;

public final class FbUtils {
  public static String convert(String[] args) {
    StringBuilder fields = new StringBuilder();

    for (String arg : args) {
      fields.append(arg).append(',');
    }

    return removeLastComma(fields.toString());
  }

  private static String removeLastComma(String str) {
    str = str.trim();

    if (str.length() > 0 && str.charAt(str.length() - 1) == ',') {
      str = str.substring(0, str.length() - 1);
    }

    return str;
  }
}

File: libs/api/src/main/kotlin/com/example/api/graphapi/utils/FbUtils.kt
package com.example.api.graphapi.utils

object FbUtils {
  fun convert(args: Array<String>): String {
    val fields = StringBuilder()

    for (arg in args) {
      fields.append(arg).append(',')
    }

    return removeLastComma(fields.toString())
  }

  private fun removeLastComma(value: String): String {
    var str = value
    str = str.trim { it <= ' ' }

    if (str.isNotEmpty() && str[str.length - 1] == ',') {
      str = str.substring(0, str.length - 1)
    }

    return str
  }
}
